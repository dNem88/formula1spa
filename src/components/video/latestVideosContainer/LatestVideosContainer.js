import React from 'react'
import styles from './latestVideosContainer.module.css'
import VideosList from '../VideosList/VideosList'

function LatestVideosContainer(props) {
    return (
        <section className={styles['main-container']}>
            <VideosList {...props}/>
        </section>
    )
}

export default LatestVideosContainer