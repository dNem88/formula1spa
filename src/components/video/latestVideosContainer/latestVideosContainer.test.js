import React from 'react'
import {MemoryRouter} from 'react-router-dom'
import renderer from 'react-test-renderer'
import LatestVideosContainer from './LatestVideosContainer'


jest.mock('../VideosList/VideosList', () => () => 'VideosList')

describe('Testing LatestVideosContainer', () => {
    
    it('Render Properly', () => {
        const snapshot = renderer.create(
        <MemoryRouter>
            <LatestVideosContainer/>
        </MemoryRouter>
    ).toJSON()
        expect(snapshot).toMatchSnapshot()
        expect(snapshot.type).toBe('section')
        expect(snapshot.props.className).toBe('main-container')
    })
})