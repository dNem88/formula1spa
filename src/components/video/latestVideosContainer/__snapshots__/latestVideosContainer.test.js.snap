// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing LatestVideosContainer Render Properly 1`] = `
<section
  className="main-container"
>
  VideosList
</section>
`;
